pkg/apis/options/options.go

```diff
@@ -60,6 +60,7 @@ type Options struct {
 	SSLInsecureSkipVerify bool     `flag:"ssl-insecure-skip-verify" cfg:"ssl_insecure_skip_verify"`
 	SkipAuthPreflight     bool     `flag:"skip-auth-preflight" cfg:"skip_auth_preflight"`
 	ForceJSONErrors       bool     `flag:"force-json-errors" cfg:"force_json_errors"`
+	EncodeState           bool     `flag:"encode-state" cfg:"encode_state"`
```

Comment:
@tuunit should this also be added to the alpha options?

@brezinajn I think this also should be added to the documentation.

Reply:
Documented [here](https://github.com/oauth2-proxy/oauth2-proxy/pull/2239/files#diff-51f65c5c3d17696023478be995d41a1af64cb188449c938d057f029787b3af3bR214)

Reply:
@kvanzuijlen Yes and no. Unfortunately, this complete block isn't yet supported by new AlphaConfig. That is an open issue to be refactored to fully support all server, proxy and core settings inside the AlphaConfig because they currently only available as flags or via the legacy config.